sql.txt

```diff
@@ -1,14 +1,39 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country
+FROM Customers
+ORDER BY Country
+LIMIT 10 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT *
+FROM Customers
+WHERE ContactName LIKE "A%";
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT *
+FROM OrderDetails
+WHERE ProductID = 51 AND Quantity > 10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price)
+VALUES ("Maple Syrup", 1, 1, "4 - 16oz bottles", 13),
+("Sunshine Spice", 2, 2, "3 - 12oz jars", 4),
+("Budweiser", 16, 1, "8 - 12oz bottles", 14);
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+UPDATE Products SET Price = + 1.50
+WHERE ProductID = 1 AND ProductID = 2;
```

Comment:
Mind your ANDs and ORs! In this case, we don't want a product with both ids of 1 AND 2 (which does not exist), but rather OR.